app/app.go

```diff
@@ -204,12 +210,14 @@ var (
 		erc20types.ModuleName:          {authtypes.Minter, authtypes.Burner},
 		claimstypes.ModuleName:         nil,
 		incentivestypes.ModuleName:     {authtypes.Minter, authtypes.Burner},
+		feestypes.ModuleName:           {authtypes.Minter, authtypes.Burner},
 	}
 
 	// module accounts that are allowed to receive tokens
 	allowedReceivingModAcc = map[string]bool{
 		distrtypes.ModuleName:      true,
 		incentivestypes.ModuleName: true,
+		feestypes.ModuleName:       true,
```

Comment:
I don't think the module account needs to receive tokens from external accounts. Do you see a use case for that?

Reply:
EDIT: I think a module account is not required?